### DISCUSSION TITLE
General questions about PR reviews for new formulas

### DISCUSSION THREAD

**Discussion by andreaangiolillo** [Homebrew/discussions#3143] [Writing Formulae/Casks] [answered]
Hello everyone 👋 ,

MongoDB has recently released a new open-source product called [AtlasCLI](https://www.mongodb.com/docs/atlas/cli/stable/) that we would love to add to homebrew-core. I submitted a PR about 2 weeks ago but haven’t received a response. Is there anything I can do to help progress the review or help in any way?

Do you have a rough timeline for how long it takes to get a new formula added normally? It would really help to know the timeline so that we can schedule additional works on our side to add brew to our documentation.

Thank you very much!

**Comment by SMillerDev** [comment, marked as answer]
It'll be done when a volunteer has time and motivation to review it

**Reply by andreaangiolillo** [reply]
got it. Thank you very much for the quick response 🙇‍♂️
